client/dashboard/domains/forwarding/index.tsx

```diff
@@ -0,0 +1,193 @@
+import { useSuspenseQuery, useMutation } from '@tanstack/react-query';
+import { Link, useRouter } from '@tanstack/react-router';
+import { Button } from '@wordpress/components';
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useState, useMemo } from 'react';
+import {
+	domainForwardingDeleteMutation,
+	domainForwardingQuery,
+} from '../../app/queries/domain-forwarding';
+import { domainRoute, domainForwardingAddRoute, domainForwardingEditRoute } from '../../app/router';
+import DataViewsCard from '../../components/dataviews-card';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+import type { DomainForwardingObject } from '../../data/domain-forwarding';
+import type { Action, Field, ViewTable, ViewList, View } from '@wordpress/dataviews';
+
+function getForwardingId( forwarding: DomainForwardingObject ) {
+	return `${ forwarding.domain_redirect_id }-${ forwarding.domain }`;
+}
+
+type ForwardingView = ViewTable | ViewList;
+
+const DEFAULT_VIEW: ForwardingView = {
+	type: 'table',
+	search: '',
+	page: 1,
+	perPage: 20,
+	titleField: 'source',
+	sort: {
+		field: 'source',
+		direction: 'asc',
+	},
+	fields: [ 'destination' ],
+	filters: [],
+};
+
+const DEFAULT_LAYOUTS = {
+	table: {},
+	list: {},
+};
+
+function DomainForwarding() {
```

Comment:
Let's rename the directory to `domains/overview-forwarding`, to be consistent with `overview-dnssec` (since it's part of the Overview tab)

Reply:
It looks to be part of the Overview navigation? See the breadcrumbs in the Figma design:

<img width="254" height="95" alt="image" src="https://github.com/user-attachments/assets/c7e77b1c-1e1e-4bb6-9f98-c36f16c1edfd" />

In some sense it's part of Overview 😄 

If that feels weird, maybe we can replace the proposed `overview-` prefix with simply `domain-`. So, `/domain-dnssec`, `/domain-forwarding`.

Reply:
okay, that makes more sense